examples/README.md

```diff
@@ -0,0 +1,113 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── properties.yml
+│   ├── autograde.tar
+│   └── autograde-Makefile
+└── assessment2
+    └── properties.yml
+
+```
+
+every folder in the tar represents a separate assessment
+
+assessment1 is an example of an assessment with an autograding component
+
+properties.yml
+```YAML
+---
+general:
+  name: assessment1
+  description: ''
+  display_name: ASSESSMENT WITH AUTOGRADING
+  handin_filename: handin.zip
+  handin_directory: handin
+  max_grace_days: 0
+  handout: ''
+  writeup: ''
+  max_submissions: -1
+  disable_handins: false
+  max_size: 2
+  has_svn: false
+  category_name: Lab Activities
+handin:
+  start_at: '1523557920'
+  visible_at: '1523557967'
+  due_at: '1543692720'
+  end_at: '1543699920'
+  grading_deadline: '1543693720'
+problems:
+- name: Score
+  description: ''
+  max_score: 3.0
+  optional: false
+autograder:
+  autograde_timeout: 180
+  autograde_image: autograding_image
+  release_score: true
+  makefile: autograde-Makefile
+  tarfile: autograde.tar
+book_keeping:
+  interface_number: 1000
```

Comment:
This is a nit, but I would really just make this something like `version: 1` at the top level.

Sorry I missed it before, I must have been going too fast.  I even saw the interface 1000 in the Ruby, but didn't put 2 & 2 together.

Reply:
changed